src/digraphs.c

```diff
@@ -896,24 +905,24 @@ static Obj FuncDIGRAPH_SYMMETRIC_SPANNING_FOREST(Obj self, Obj adj) {
       stack[1] = 1;
 
       while (1) {
-        j = stack[0];
-        k = stack[1];
+        j   = stack[0];
+        k   = stack[1];
         nbs = ELM_PLIST(adj, j);
 
         // idea: is <nbs[k]> a new vertex? add edges <j> <-> <nbs[k]> if so
 
         // if we're finished with <j>, or <nbs[k]> doesn't exist, then backtrack
         if (ptr[j] || k > (UInt) LEN_LIST(nbs)) {
-          ptr[j] = 1; // vertex <j> and its descendents are now fully explored
+          ptr[j] = 1;  // vertex <j> and its descendents are now fully explored
```

Comment:
And this here – it would be nice to know what style guide you're going by so that we can all be using it.